ndcube/tests/helpers.py

```diff
@@ -61,3 +63,15 @@ def assert_wcs_are_equal(wcs1, wcs2):
     assert list(wcs1.wcs.cdelt) == list(wcs2.wcs.cdelt)
     assert list(wcs1.wcs.cunit) == list(wcs2.wcs.cunit)
     assert wcs1.wcs.naxis == wcs2.wcs.naxis
+
+
+def create_sliced_wcs(wcs, item, dim):
+	"""
+	Creates a sliced `SlicedFITSWCS` object from the given slice item
+	"""
+
+	# Sanitize the slices
+	item = sanitize_slices(item, dim)
```

Comment:
does this need reversing here? if not why are you sanitizing it before passing it in?

Reply:
Nope, just that the format of `item` does not create issues, so I used `sanitize_slices`.
Does `sanitize_slices` reverse the order?

Reply:
> Does sanitize_slices reverse the order?

No.

But `SlicedLowLevelWCS` calls `sanitize_slices` itself, so I don't think we need to do it explicitly. (Although I do wonder about correct ordering here).